source/help/getting-started/managing-members.rst

```diff
@@ -1,77 +1,97 @@
 
-#### Member 
+Member
+~~~~~~
 
 This is the default role given to users when they join a team. Members have basic permissions on the Mattermost team.
 
-#### Team Admin 
+Channel Admin
+~~~~~~~~~~~~~
+
+The person who creates a channel is assigned the Channel Admin role for that channel. People with the Channel Admin role have the following privileges:
+
+- Ability to assign the Channel Admin role to other members of the channel.
+- Ability to remove the Channel Admin role from other holders of the Channel Admin role.
+- Ability to remove members from the channel.
 
-When a team is first created, the person who set it up is made a Team Administrator. It is a team specific role, meaning that someone can be a Team Admin for one team but only a Member on another team. Team Administrators have the following privileges: 
+Team Admin
+~~~~~~~~~~
+
+When a team is first created, the person who set it up is made a Team Administrator. It is a team specific role, meaning that someone can be a Team Admin for one team but only a Member on another team. Team Administrators have the following privileges:
 
 - Access to the "Team Settings" menu from the team site main menu
 - Ability to change the team name and import data from Slack export files
-- Access to the "Manage Members" menu, where they can control whether team members are "Members" or "Team Administrators" 
+- Access to the "Manage Members" menu, where they can control whether team members are "Members" or "Team Administrators"
 
-#### System Admin
+System Admin
+~~~~~~~~~~~~
 
 The first user added to a newly installed Mattermost system is assigned the System Admin role.
 
-The System Admin is typically a member of the IT staff and has all the privileges of a Team Admin, along with the following additional privileges: 
+The System Admin is typically a member of the IT staff and has all the privileges of a Team Admin, along with the following additional privileges:
 
 - Access to the System Console from the main menu in any team site
 - Ability to change any setting on the Mattermost server available in the System Console
 - Ability to promote and demote other users to and from the System Admin role
 - Ability to make a user account Inactive and to reactivate an Inactive account
 
-#### Inactive 
+Inactive
+~~~~~~~~
 
-A System Administrator has the ability to make a user account Inactive by going to **System Console > Teams**, and selecting a team to manage. From there they can go to the **User** page for the team, and manage a user's role. 
+A System Administrator has the ability to make a user account Inactive by going to **System Console > Teams**, and selecting a team to manage. From there they can go to the **User** page for the team, and manage a user's role.
```

Comment:
I think the System Admin is the role, but the System Administrator is the person who has the System Admin role. So you can say either "The System Administrator can ..." or "Anyone with the System Admin role can ..."